radiometric_normalization/usage.py

```diff
@@ -51,6 +51,9 @@ def perform_data_process(image_path, ref_image_path=None, out_directory=None, ou
     region = image_name.split('_')[0]
     candidate_path = image_path
     reference_path = None
+    ref_meta_path = None
```

Comment:
Were these variables necessary?

Reply:
fixed in 66eb6f71ecb7fb73af77ea53da97d93eba6ab27c